test/fixtures/schedule-sidebar-state.js

```diff
@@ -23,13 +23,13 @@ const suppliers = [
     status: TO_BE_PUBLISHED,
   },
   {
-    name: "Supplier in Contentful 3",
+    name: "Supplier not in Contentful",
     rank: "2",
     overallRating: "3.9",
     complaintsRatings: "2.9",
     contactRating: "1.9",
     guaranteeRating: "0.9",
-    id: 3,
+    id: 4,
```

Comment:
Could be a silly question - why could we not keep the id of 3? Is it because it was already used for the "Supplier in Contentful 3" and is no longer available?